### ISSUE TITLE
(Feature Flaw) CreditScore should be more robust

### ISSUE THREAD

**Issue by Leeyp** [Leeyp/pe#3]
Issue 1: Person who has loaned money for a very long time or did not even pay back should not have a positive credit score:

![image.png](https://raw.githubusercontent.com/Leeyp/pe/main/files/7d03bb50-567c-456f-85ab-2a6f16118a0a.png)

Issue 2: Person who has loaned a very large amount of money but did not pay back should not even have a positive credit score (still 50?)

![image.png](https://raw.githubusercontent.com/Leeyp/pe/main/files/05e47e65-1401-4e5a-9d81-f0489acff9aa.png)

TL;DR 
Creditscore is an insufficient feature to judge actual loan/credit rating because it does not take into account 1. (unreasonably long) length of return of loan, 2. if the person has paid back loan at all, or 3. amount of loan.
